test/spec.yml

```diff
@@ -227,7 +293,7 @@
              listtest2                *201*-*-*T*:*:*Z*
     -   in: bn list --json
         out: |-
-            *[*{"name":"listtest1","lastDeployed":"[20*-*-*T*:*:*Z]"}*,*{"name":"listtest2","lastDeployed":"[20*-*-*T*:*:*Z]"}*]*
+            *[*{"name":"listtest1","lastDeployed":"20*-*-*T*:*:*Z"}*,*{"name":"listtest2","lastDeployed":"20*-*-*T*:*:*Z"}*]*
```

Comment:
Again, these are the worst offenders in cases like #393.  Please use `#` and `%`.

Reply:
done in d78d791